environment.yml

```diff
@@ -5,7 +5,8 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - pytorch=1.12
+  - tensorflow
+  - pytorch
```

Comment:
I believe pytorch 2 will break parts of armory; is that restriction asserted anyplace else?

Reply:
Updated pinned torch version in `pyproject.toml` to `torch >=1.12.0,<=1.13.1`.